packages/dbos-cloud/databases/databases.ts

```diff
@@ -321,3 +323,57 @@ export async function restoreUserDB(host: string, dbName: string, targetName: st
     return 1;
   }
 }
+
+export async function connect(host: string, dbName: string, password: string) {
+  const logger = getLogger();
+
+  try {
+    if(!existsSync(dbosConfigFilePath)) {
+      logger.error(`Error: ${dbosConfigFilePath} not found`);
+      return 1;
+    }
+    logger.info(`Backing up ${dbosConfigFilePath} to ${dbosConfigBackupPath}`)
+
+    copyFileSync(dbosConfigFilePath, dbosConfigBackupPath);
+
+    logger.info("Retrieving cloud database info...");
+    const userDBInfo = await getUserDBInfo(host, dbName);
+    console.log(`Postgres Instance Name: ${userDBInfo.PostgresInstanceName}`);
+    console.log(`Host Name: ${userDBInfo.HostName}`);
+    console.log(`Port: ${userDBInfo.Port}`);
+    console.log(`Database Username: ${userDBInfo.DatabaseUsername}`);
+    console.log(`Status: ${userDBInfo.Status}`);
+
+    logger.info(`Loading cloud database connection information into ${dbosConfigFilePath}...`)
+    const config: ConfigFile = loadConfigFile(dbosConfigFilePath);
+    config.database.hostname = userDBInfo.HostName;
+    config.database.port = userDBInfo.Port;
+    config.database.username = userDBInfo.DatabaseUsername;
+    config.database.password = password;
+    writeConfigFile(config, dbosConfigFilePath);
+    logger.info(`Cloud database connection information loaded into ${dbosConfigFilePath}`)
+    return 0;
+  } catch (e) {
+    const errorLabel = `Failed to retrieve database record ${dbName}`;
+    const axiosError = e as AxiosError;
+    if (isCloudAPIErrorResponse(axiosError.response?.data)) {
+      handleAPIErrors(errorLabel, axiosError);
+    } else {
+      logger.error(`${errorLabel}: ${(e as Error).message}`);
+    }
+    return 1;
+  }
+}
+
+export function disconnect() {
+  const logger = getLogger();
+
+  logger.info(`Restoring ${dbosConfigFilePath} from ${dbosConfigBackupPath}`)
+  if(!existsSync(dbosConfigBackupPath)) {
+    logger.error(`Error: ${dbosConfigBackupPath} not found`);
+    return 1;
+  }
+  copyFileSync(dbosConfigBackupPath, dbosConfigFilePath);
+  unlinkSync(dbosConfigBackupPath);
```

Comment:
This is a bit flaky. We have no idea that `dbosConfigBackupPath` is still what it was when we created it during `connect`. Why not just drop `disconnect` altogether? If that's something really useful we can revisit

Reply:
Good point, it's needlessly complex, removed it.